content/en/docs/setup/platform-setup/kubesphere/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: KubeSphere Container Platform
+description: Instructions to setup a KubeSphere Container Platform for Istio.
+weight: 18
+skip_seealso: true
+aliases:
+    - /docs/setup/kubernetes/prepare/platform-setup/kubesphere/
+    - /docs/setup/kubernetes/platform-setup/kubesphere/
+keywords: [platform-setup,kubesphere,kubernetes]
+---
+
+Follow these instructions to prepare an open source [KubeSphere Container Platform](https://github.com/kubesphere/kubesphere) for Istio. You can download KubeSphere to easily install a Kubernetes cluster to your Linux machines.
+
+{{< tip >}}
+KubeSphere provides [All-in-One](https://kubesphere.io/docs/v2.1/en/installation/all-in-one/) and [Multi-Node](https://kubesphere.io/docs/v2.1/en/installation/multi-node/) installation option, enables you to quickly set up and manage Kubernetes and Istio in a unified web console. This tutorial will only walk you through the All-in-One installation, see [Multi-node Installation](https://kubesphere.io/docs/v2.1/en/installation/multi-node/) for further information.
+{{< /tip >}}
+
+## Prerequisites
+
+A new Linux machine (64 bit), supports VM or BM, at least 2 CPUs and 4 GB of memory, currently it supports following OS:
+
+
+- CentOS 7.4 ~ 7.7
+- Ubuntu 16.04/18.04 LTS
+- RHEL 7.4
+- Debian Stretch 9.5
+
+
+{{< tip >}}
+We suggest you to disable and stop the firewall, instead, if your network configuration uses an firewall, you must ensure it meets the [port requirements](https://kubesphere.io/docs/v2.1/en/installation/port-firewall/).
+{{< /tip >}}
+
+## Provisioning Kubernetes Cluster
+
+1. Download KubeSphere to your Linux machine, it can help you to create a standard Kubernetes cluster (v1.15 by default, it also supports v1.13 and v1.14).
```

Comment:
The word `it` references K8s, or KubeSphere, cluster, machine, or Linux? Any time you consider using the word it, instead I recommend putting the noun in instead.